### PR TITLE
fix: clean up experiment winner for scroll block

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Cleanup experiment winner for scroll block
- Left analytics intact with winner value

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1306 #done

### CODE REVIEW

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx [hunk 1]
```diff
@@ -87,38 +39,23 @@ export default function ScrollFeedFiltersOnboarding({
       }}
     >
       <div className="flex flex-col items-center">
-        {version === ScrollOnboardingVersion.V1 ? (
-          <GaussianBlur version={version} />
-        ) : (
-          <div className="flex relative justify-center items-center mb-10">
-            <GaussianBlur version={version} />
-            <SuperchargeIcon className="w-[4.625rem] h-[4.625rem]" />
-          </div>
```
Comment (sshanzel) [inline packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:96]: Should this be still rendered? Looking at the condition it renders on v2.

**Review by sshanzel — CHANGES_REQUESTED** [review]
Nice clean up! Just had one question regarding some elements to render.

File: packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx:96]: Oh totally missed this, it should indeed.
Thought it was a single render at first.

**Review by sshanzel — APPROVED** [review]
Let's do it! 🥳

**Review by capJavert — APPROVED** [review]
🚀

**Pull request merged**